PyStemmusScope/soil_io.py

```diff
@@ -0,0 +1,229 @@
+from pathlib import Path
+import hdf5storage
+import numpy as np
+import xarray as xr
+from . import variable_conversion as vc
+
+
+def _open_multifile_datasets(paths, lat, lon, lat_key='lat', lon_key='lon'):
```

Comment:
I couldnot understand why xarray does not support this. After installing `dask`, I tested the code below where `paths=ptf_files`:

```python
ds = xr.open_mfdataset(paths, combine='by_coords')
ds = ds.sel({lat_key: lat, lon_key: lon}, method='nearest')
```

We can add "[dask](https://docs.dask.org/en/stable/install.html)" to the setup requirement. it will be used in the future.

Reply:
I implemented it this way, as it is a lot more efficient in the way we are using it. Using dask and `xr.open_mfdataset` is extremely slow for how we are using it right now.  The files are many gigabytes, and using xarray's mfdataset, while simple, will take a long time to run.
`open_mfdataset` does allow for a preprocessor, but we cannot pass arguments to this preprocessor.

Of course if we want to run the model globally it can make sense, but if you are only interested in getting data for a larger amount of locations.